config/environment.js

```diff
@@ -30,33 +30,32 @@ const envvars = {
  * @description
  * Browser list for autoprefixer, see https://github.com/ai/browserslist.
  * Support output lists can be viewed via copy/pasting into http://browserl.ist.
- * @param {string} codeType - The code to support. Either 'js' or 'css'.
+ * @param {string} task - The gulp task calling for it's associated browsers.
```

Comment:
nix gulp if you're going agnostic here, e.g. webpack uses this too.

Reply:
Fixed.

Reply:
*its